palette/tests/convert/lab_lch.rs

```diff
@@ -7,8 +7,8 @@ fn lab_lch_green() {
     let expect_lab = lch.into_lab();
     let expect_lch = lab.into_lch();
 
-    assert_relative_eq!(lab, expect_lab, epsilon = 0.001);
-    assert_relative_eq!(lch, expect_lch, epsilon = 0.001);
+    assert_relative_eq!(lab, expect_lab, epsilon = 0.1);
```

Comment:
Have you tried to tighten these error margins again?

Reply:
I have -- it works fine :smiley: thanks to `libm`.
I'll push those changes in a moment.

Reply:
Great! :smile: